assets/src/application/services/hooks/useTimeZoneTime.ts

```diff
@@ -2,18 +2,22 @@ import { path } from 'ramda';
 import { parseISO, isValid } from 'date-fns';
 
 import useConfig from '../config/useConfig';
-import { localToUtc, utcToLocal } from '../utilities/date';
```

Comment:
might as well delete these functions from here since they are not used anywhere else and are kinda useless without the timezone, which basically means you would have to duplicate the logic in `useTimeZoneTime.ts` in order to use the functions, which would be pointless.